### DISCUSSION TITLE
How do I stop seeing feedback from some users?

### DISCUSSION THREAD

**Discussion by leoheck** [community/community#27762] [New to GitHub]
Title says it all.

**Comment by NikhilMath** [comment]
what do you mean?
Can you block some users?

**Reply by leoheck** [reply]
Why would I block some users? I just don't want to see feedback and shit from other users. Just like normal or real life where you go home so you don't need to see your neighbor every time. Or when you use headphones to avoid listening to the shitty music that comes from another one at the next table. Or when you go to the library to read things in silence so you can have your own ideas instead of being pushed by shit that you could care less. But why would I block anyone? If someone still needs to talk to you, he can reach out to you to say something. Like, hey dude, your house is catching fire. Why whole internet thing is all about blocking and jailing people to death? I just don't want to have a TV Advertising every shit that happens around me E.V.E.R.Y.T.I.M.E. Did you get it?

**Comment by radar-blip** [comment]
Just bookmark your profile, or click on your profile as soon as you sign on. 
That way you dont have to see all that stuff.
